code/src/ColorQuestion.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react' 
+
+export const ColorQuestion = ({ color, onColorChange}) => {
```

Comment:
Another way to export (that you probably already know about) when you only have one function in the code to export is to use the un-named export `export default ColorQuestion` which could be placed at the end of the file instead. But this works too :)